packages/web-twig/tests/components-fixtures/selectValidationState.twig

```diff
@@ -0,0 +1,36 @@
+<Select
```

Comment:
It would be great if we could keep the same structure as in `packages/web-twig/tests/components-fixtures/fileUploaderInput.twig`, but do as you want